streampark-flink/streampark-flink-client/streampark-flink-client-api/src/main/scala/org/apache/streampark/flink/client/bean/DeployResponse.scala

```diff
@@ -17,4 +17,11 @@
 
 package org.apache.streampark.flink.client.bean
 
-case class DeployResponse(address: String, clusterId: String)
+import javax.annotation.Nullable
+
+import java.util.{Map => JavaMap}
+
+case class DeployResponse(
+    @Nullable address: String = "",
+    clusterId: String,
+    flinkConfig: JavaMap[String, String])
```

Comment:
Can you tell me why we added this field? Where do we use it?

Reply:
I am also very confused about this part, mainly to align submitrequest, I do not know how submitrquest uses this parameter

Reply:
The flinkconfig parameter is very important during the deployment process

Reply:
> The flinkconfig parameter is very important during the deployment process

I see that we don't currently need this parameter, so I think it would be more appropriate to add it when we actually need it.

Reply:
ok

Reply:
done